lib/cp8_cli/global_config.rb

```diff
@@ -16,6 +17,29 @@ def github_token
 
       attr_reader :store
 
+      def initialize_store
+        migrate_legacy_store if uses_legacy_store?
+
+        default_store
+      end
+
+      def uses_legacy_store?
+        !default_store.exist? && legacy_store.exist?
```

Comment:
Super Nit: Maybe would be enough to check for the presence of the legacy store here, assuming that the migration gets rid of it?

```suggestion
        legacy_store.exist?
```

Reply:
Ah, yeah, I imagine the presence of both legacy and default stores is something that is not supposed to happen _within normal use of this gem_. 👌🏻